### DISCUSSION TITLE
NNCF could not compile GPU quantizaiton extensions

### DISCUSSION THREAD

**Discussion by siyhust** [openvinotoolkit/nncf#2712] [Q&A]
I‘m using nncf 2.6.0 with openvino 2023.3，torch 2.0.1+cu117, ans I installed all packages with pip.

It's ok when I using cpu for PTQ with nncf.quantize，but when I  using GPU, it throw a bug below:

"RuntimeError: CUDA is available for pytorch, but NNCF could not compile GPU quantization extensions. Make sure that you have installed CUDA development tools……"

But I indeed install CUDA on  /usr/local/cuda-11.7,  I also set enviroment with add CUDA path to os.environ['PATH'].

Hope some kind person help me with this question, thanks a lot~

**Comment by siyhust** [comment]
Is it the reason that I’m using a torch version too low for NNCF ?

**Comment by MaximProshin** [comment]
@AlexanderDokuchaev , @daniil-lyakhov , please take a look.

**Comment by AlexanderDokuchaev** [comment]
Hi @siyhust 
Version of NNCF an PyTorch are ok, you can find recommended versions [here](https://docs.nvidia.com/cuda/cuda-installation-guide-linux/index.html). 

Please check that you have installed `nvcc` compiler by `nvcc --version` command, may be you have installed only runtime part of cuda.

**Comment by siyhust** [comment]
@AlexanderDokuchaev 
I found it runs ok when I use native terminal instead of Pycharm
Problem fixed by founding correct way to set  PATH  in my Pycharm.
Thanks a lot.
